.travis.yml

```diff
@@ -56,6 +56,19 @@ jobs:
     - GEMSETS="test rubocop"
     rvm: 2.4.1
     script: bundle exec ./bin/rubocop lib/
+  - stage: test
+    addons:
+      apt:
+        packages:
+        - docker-ce
+    sudo: required
+    sevices:
+    - docker
+    env:
+    - BUNDLE_VERSION=1.12.0
+    - GEMSETS="test maze"
+    rmv: 2.4.1
+    script: bundle exec bugsnag-maze-runner features/plain_features
```

Comment:
Do we only want to run this particular feature on CI, or are there others?

Reply:
I'm splitting it into jobs, as the 50minute limit is a concern if the plain and rails tests were to run serially

Reply:
I'll update `.travis.yml` in the rails PR to add that job